README.md

```diff
@@ -1,40 +1,181 @@
-# Product Details
+# VTEX Product Details
 
-Product Details is a canonical component that any VTEX store can use.
+## Description
+The VTEX Category Menu app shows the details of a product like image, name and price. THis app is used by store theme.
+
+:loudspeaker: **Disclaimer:** Don't fork this project. Use, contribute, or open issue with your feature request.
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
-| [0.x]    | **Maintenance LTS** |  2018-05-29     | 2018-11-28            | March 2019  | 1.x
 | [1.x]    | **Current Release** |  2018-11-28     |                       |             | 2.x
+| [0.x]    | **Maintenance LTS** |  2018-05-29     | 2018-11-28            | March 2019  | 1.x
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
+
+## Table of Contents
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+    - [CSS namespaces](#css-namespaces)
+- [Troubleshooting](#troubleshooting)
+- [Tests](#tests)
 
 ## Usage
 
-To use it, you need to add the dependency in your `manifest.json`
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
```

Comment:
Didn't we settled with `Store Builder`? I can't really remember

Reply:
It was only the app name, or am I mistaken?